code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -77,7 +77,11 @@
 /datum/reagent/consumable/nutriment/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
 	. = ..()
 	if(SPT_PROB(30, seconds_per_tick))
-		if(affected_mob.heal_bodypart_damage(brute = brute_heal * REM * seconds_per_tick, burn = burn_heal * REM * seconds_per_tick, updating_health = FALSE, required_bodytype = BODYTYPE_ORGANIC))
+		var/healing_modifier = 1
+		var/obj/item/organ/earth_pony_core/earth_core = affected_mob.get_organ_by_type(/obj/item/organ/earth_pony_core) // Earth Ponies heal better from food
+		if (earth_core)
+			healing_modifier += earth_core.maxHealth / (earth_core.maxHealth - earth_core.damage)
```

Comment:
this formula is wrong, I think. it currently gives massive boosts to healing_modifier if your core is near-destruction, if I'm reading it right.

Reply:
whoops, good catch, I'll fix that